tests/Controller/EnMarche/MembershipControllerTest.php

```diff
@@ -50,11 +50,11 @@ public function testCannotCreateMembershipAccountWithSomeoneElseEmailAddress($em
         $data = static::createFormData();
         $data['user_registration']['emailAddress']['first'] = $emailAddress;
         $data['user_registration']['emailAddress']['second'] = $emailAddress;
-        $crawler = $this->client->submit($crawler->selectButton('Créer mon compte')->form(), $data);
+        $crawler = $this->client->submit($crawler->selectButton('J\'adhère')->form(), $data);
```

Comment:
`Adhérer` we've started using infinitive for call to actions (CTAs) in favor of first person.